### ISSUE TITLE
Allow torch backend when computing partial wasserstein plan

### ISSUE THREAD

**Issue by jules-samaran** [PythonOT/POT#456]
Hi, 
it seems that unlike ot.lp.emd for example, functions in ot.partial don't allow inputs to be torch tensors but the error message isn't very explicit. See the following example where the last line raises an error:

```
import ot
import torch 
import numpy as np

C = np.random.rand(3, 4)
p = ot.unif(C.shape[0], type_as=C)
q = ot.unif(C.shape[1], type_as=C)
mass = 0.8
plan_np = ot.partial.partial_wasserstein(p, q, C, m=mass)

plan_torch = ot.partial.partial_wasserstein(torch.tensor(p), torch.tensor(q), torch.tensor(C), m=mass)
```

Is this on purpose?

**Comment by rflamary** [comment]
Hello I believe that has been fixed in the new release (0.9.0) from last friday. I do not have a bug anymore could you please check?

**Comment by jules-samaran** [comment]
Thanks for your answer. I had forgotten to check the release, it works now!
